src/taskgraph/util/treeherder.py

```diff
@@ -62,3 +62,30 @@ def inherit_treeherder_from_dep(job, dep_job):
     # Does not set symbol
     treeherder.setdefault("kind", "build")
     return treeherder
+
+
+def treeherder_defaults(kind, label):
+    default_collection = "opt"
+    label_without_kind = label.split(f"{kind}-", 1)[1]
```

Comment:
Are we sure that the label starts with `{kind}-`, or could we end up with an IndexError here?

Reply:
Good question, I will try to verify this assumption.

Reply:
Labels don't have to start with `{kind}-`, that's only a convention. I am positive we have several places that break this convention.